shared/teams/channel/header.tsx

```diff
@@ -37,6 +39,13 @@ const HeaderTitle = (props: HeaderTitleProps) => {
         path: [{props: {conversationIDKey, teamID}, selected: 'chatEditChannel'}],
       })
     )
+  const onAddMembers = () =>
+    dispatch(
+      nav.safeNavigateAppendPayload({
+        // TODO: this route does not exist yet
```

Comment:
Is this going to cause a black bar? If so, maybe just do a `() => console.log('onAddMembers not implemented yet')`

Reply:
It does not cause a black bar.